scala/lambdas/ingest-mapper/src/test/scala/uk/gov/nationalarchives/ingestmapper/MetadataServiceTest.scala

```diff
@@ -115,7 +115,7 @@ class MetadataServiceTest extends AnyFlatSpec with TableDrivenPropertyChecks {
           val input = Input(batchId, URI.create("s3://bucket/prefix/metadata.json"), "executionName")
 
           def createCollectionAsset(obj: Obj) =
-            Option(DiscoveryCollectionAsset(obj("name").str, DiscoveryScopeContent(obj("description").str), Option(obj("title").str)))
+            Option(DiscoveryCollectionAsset(obj("name").str, DiscoveryScopeContent(Option(obj("description").str)), Option(obj("title").str)))
```

Comment:
1. Shouldn't we use `strOpt` for title and description, like `Option(DiscoveryCollectionAsset(obj("name").str, DiscoveryScopeContent(obj("description").strOpt), obj("title").strOpt))`, since the values might not be in the table?
2. If the title and description are optional, should we have a test for when they are not in the table?

Reply:
1. Yeah, good idea. 
2. There is a test already in DiscoveryServiceTest which checks that table/description aren't output when they're empty and I think that covers all wel need.